pasqal_cloud/client.py

```diff
@@ -170,7 +170,7 @@ def _send_workload(self, workload_data: Dict[str, Any]) -> Dict[str, Any]:
 
     def _get_workload(self, workload_id: str) -> Dict[str, Any]:
         workload: Dict[str, Any] = self._request(
-            "GET", f"{self.endpoints.core}/api/v1/workloads/{workload_id}"
+            "GET", f"{self.endpoints.core}/api/v2/workloads/{workload_id}"
```

Comment:
As we are now targeting a new endpoint version wouldn't it be a breaking change ? So this wouldn't be the 0.3.6 but the 0.4 ?

Reply:
A breaking change is something that requires the user or consuming component to change their use of the sdk. Here the updated endpoint is in a private function and requires no action from consuming user/component. Therefore it is not a breaking change.